connectors/katalog/pkg/connector/connector.go

```diff
@@ -121,6 +121,10 @@ func (r *Handler) createAsset(c *gin.Context) {
 		assetPrefix = request.DestinationAssetID + "-"
 	}
 
+	// Split the string after the first "/", if exists, and take what's after the "/"
+	slices := strings.SplitAfterN(assetPrefix, "/", 2)
+	assetPrefix = slices[len(slices)-1]
+
```

Comment:
I don't see how this turns an illegal asset name into a legal one for a general case.
`a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')" for field "metadata.generateName"`

Reply:
The test doesn't fail b/c internally it makes it a valid name.